etc/init.d/lncm-post

```diff
@@ -267,8 +280,8 @@ start() {
   }
 
   persist_state() {
-    echo "Backup apkovl.tar.gz to USB storage"
-    /sbin/lbu package /media/important/important/box.apkovl.tar.gz
+    echo "Backup apkovl.tar.gz to Root"
```

Comment:
"…to filesystem root (`/`)".

Reply:
Good idea might actually leave it out as its not even necessary anymore

Reply:
Removed